SampleApplications/SDK/Opc.Ua.Server/Opc.Ua.Server.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.4</TargetFramework>
+    <TargetFrameworks>net46;netstandard1.4;netstandard2.0</TargetFrameworks>
+    <DefineConstants Condition="'$(TargetFramework)' == 'netstandard2.0'">$(DefineConstants);NO_HTTPS</DefineConstants>
```

Comment:
Why turn off HTTPS support for .Net Standard 2.0?

Reply:
Kestrel 2.0 doesn't compile yet. HTTPS is an extra work item for .Net Standard 2.0

Reply:
fixed